<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>index</class>
 <widget class="QWidget" name="index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>618</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>618</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(224, 242, 255);
border: none;
font: 16px &quot;微软雅黑&quot;;

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="24,76">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="left">
     <property name="styleSheet">
      <string notr="true">
background-color:#2B579A;
text-align: left;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="leftTop">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>240</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#41A5EE;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="headImg">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>20</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 20px;
background-color: #fff;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="uerName">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>28</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#41A5EE;</string>
       </property>
       <property name="text">
        <string>用户名</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="setIcon">
       <property name="geometry">
        <rect>
         <x>208</x>
         <y>28</y>
         <width>24</width>
         <height>24</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	image: url(:/open/img/open/Setting_open_unselected.png);
	background-color: none;
}
QLabel:hover {
	image: url(:/selected/img/selected/Setting_selected.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="kp">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>104</y>
        <width>224</width>
        <height>40</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="kpButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>224</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>224</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>224</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: #93D2FF;
	border-radius: 14px;
	padding:46px
}

QPushButton:hover {
	background-color: #FFFFFF;
}
</string>
       </property>
       <property name="text">
        <string>科普聊天</string>
       </property>
      </widget>
      <widget class="QLabel" name="kpIcon">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>10</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/open/img/open/Chat_open_unselected.png);
background-color: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="pm">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>164</y>
        <width>224</width>
        <height>40</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pmButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>224</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>224</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>224</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: #93D2FF;
	border-radius: 14px;
	padding:46px
}

QPushButton:hover{
	background-color: #FFFFFF;
}
</string>
       </property>
       <property name="text">
        <string>排名</string>
       </property>
      </widget>
      <widget class="QLabel" name="pmIcon">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>10</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/open/img/open/Rank_open_unselected.png);
background-color: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="sc">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>284</y>
        <width>224</width>
        <height>40</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="scButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>224</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>224</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>224</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: #93D2FF;
	border-radius: 14px;
	padding:46px
}

QPushButton:hover{
	background-color: #FFFFFF;
}
</string>
       </property>
       <property name="text">
        <string>收藏</string>
       </property>
      </widget>
      <widget class="QLabel" name="scIcon">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>10</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/open/img/open/Collect_open_unselected.png);
background-color: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="qk">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>224</y>
        <width>224</width>
        <height>40</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="tjButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>224</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>224</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>224</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: #93D2FF;
	border-radius: 14px;
	padding:46px
}

QPushButton:hover{
	background-color: #FFFFFF;
}
</string>
       </property>
       <property name="text">
        <string>统计</string>
       </property>
      </widget>
      <widget class="QLabel" name="thIcon">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>10</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/open/img/open/Study_open_unselected.png);
background-color: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="setIcon_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>580</y>
        <width>28</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	
	image: url(:/chatting/img/Chatting/turnleft.png);
	background-color: none;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="right">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="inputFrame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>760</width>
        <height>168</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #fff;

QPushButton#send{
	background-color: #93D2FF;
}

QPushButton#send:hover{
	background-color: #fff;
}

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextEdit" name="sendEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>42</y>
         <width>760</width>
         <height>126</height>
        </rect>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-left: 10px;
padding-right: 10px;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="send">
       <property name="geometry">
        <rect>
         <x>686</x>
         <y>10</y>
         <width>64</width>
         <height>32</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#send{
	background-color: #93D2FF;
}

QPushButton#send:hover{
	background-color: #2B579A;
	color:#fff;
}

</string>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
      <widget class="QLabel" name="emoji">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>12</y>
         <width>22</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/chatting/img/Chatting/smile.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="speak">
       <property name="geometry">
        <rect>
         <x>47</x>
         <y>10</y>
         <width>24</width>
         <height>24</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/chatting/img/Chatting/microphone.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="dialogeArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>760</width>
        <height>370</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>760</width>
         <height>370</height>
        </rect>
       </property>
       <widget class="QFrame" name="robert">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>760</width>
          <height>58</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="80,0,0">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="robertImg">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 24px;
background-color: #fff;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="robertText">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #ffffff;
border-radius: 12px;
padding:4px</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string>你好，我是聊天机器人，小人</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>67</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="myUser">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>58</y>
          <width>760</width>
          <height>58</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="80,0,0">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="myUserImg">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 24px;
background-color: #fff;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="myUserText">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #93D2FF;
	border-radius: 12px;
	padding:4px</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string>你好</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="winTitle">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>760</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>-1</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:32px</string>
      </property>
      <property name="text">
       <string>科  科</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="minimizeButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>10</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/frame/img/Frame/minimize.png</normaloff>:/frame/img/Frame/minimize.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>10</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/frame/img/Frame/close.png</normaloff>:/frame/img/Frame/close.png</iconset>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>index</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>979</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minimizeButton</sender>
   <signal>clicked()</signal>
   <receiver>index</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>939</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
